boards/arm/nucleo_f411re/pinmux.c

```diff
@@ -34,7 +34,7 @@ static const struct pin_config pinconf[] = {
 	{STM32_PIN_PA8, STM32F4_PINMUX_FUNC_PA8_I2C3_SCL},
 	{STM32_PIN_PB4, STM32F4_PINMUX_FUNC_PB4_I2C3_SDA},
 #endif
-#if DT_NODE_HAS_STATUS(DT_NODELABEL(spi1), okay) || \
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(spi1), okay) && CONFIG_SPI || \
```

Comment:
I know && has higher precedence than ||, but this one might be parenthesized just for clarity.

Reply:
You're right @mbolivar-nordic , fixed.